src/efftester.ml

```diff
@@ -173,9 +173,11 @@ let efftostr ((ef, ev) : eff) = Printf.sprintf "(%B,%B)" ef ev
 let toOCaml ?(typeannot = true) term =
   let rec littoOcamlSB sb = function
     | LitUnit -> Printf.bprintf sb "()"
-    | LitInt i -> if i < 0 then Printf.bprintf sb "(%d)" i else Printf.bprintf sb "%d" i
+    | LitInt i -> if i <= 0 then Printf.bprintf sb "(%d)" i else Printf.bprintf sb "%d" i
```

Comment:
What's the reason for this particular change?

Reply:
The reason is the same as for the float change. I am just being precautious if we at some point want to add test for `-0`. I can see, however, that it might clutter the output, and we can remove it at least for now.

Reply:
The bug caused by `-0.` without parentheses was quite perplexing for me even though it turned out to be trivial.

Reply:
For floats I believe `-0.` is a valid concern, but for integer literals I don't follow you.